.github/workflows/ci.yml

```diff
@@ -7,8 +7,8 @@ on:
       - '!main'
 
 env:
-  BUILDER_VERSION: v0.7.0
-  BUILDER_SOURCE: releases
+  BUILDER_VERSION aws-lc
```

Comment:
is this valid yaml?

Reply:
nope, both you and GitHub spotted that :P